ee/server/api/licenses.ts

```diff
@@ -20,6 +21,7 @@ API.v1.addRoute('licenses.requestSeatsLink', { authRequired: true }, {
 		}
 
 		const url = getSeatsRequestLink();
+		Promise.await(Analytics.saveSeatRequest());
```

Comment:
Not sure this should go here... This endpoint is called every time the Users page is accessed, and not when clicking the link...